<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>620</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>710</width>
    <height>620</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>710</width>
    <height>620</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 235, 235, 206), stop:0.35 rgba(255, 188, 188, 80), stop:0.4 rgba(255, 162, 162, 80), stop:0.425 rgba(255, 132, 132, 156), stop:0.44 rgba(252, 128, 128, 80), stop:1 rgba(255, 255, 255, 0));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>131</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="cameraButton">
       <property name="text">
        <string>开启摄像头</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selectTargetButton">
       <property name="text">
        <string>选取目标物</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetStartPointButton">
       <property name="text">
        <string>起始点重置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startMeasureButton">
       <property name="text">
        <string>开始测量</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopMeasureButton">
       <property name="text">
        <string>停止测量</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="dataChartButton">
       <property name="text">
        <string>数据统计图表</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="displayGroupBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>541</width>
      <height>591</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(225, 225, 225, 255), stop:1 rgba(100, 100, 100, 255));</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QGroupBox" name="camViewDisplayGroupBox">
       <property name="minimumSize">
        <size>
         <width>517</width>
         <height>280</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>517</width>
         <height>280</height>
        </size>
       </property>
       <property name="title">
        <string>摄像机图像</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="camViewLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="dataChartGroupBox">
       <property name="minimumSize">
        <size>
         <width>517</width>
         <height>280</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>517</width>
         <height>280</height>
        </size>
       </property>
       <property name="title">
        <string>位移实时测绘图表</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QVBoxLayout" name="dataChartLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="settingButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
   </widget>
  </widget>
  <action name="actionopen_cam">
   <property name="text">
    <string>open cam</string>
   </property>
  </action>
  <action name="actionclose_cam">
   <property name="text">
    <string>close cam</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
